aoconfigpanel.cpp

```diff
@@ -221,6 +218,18 @@ void AOConfigPanel::on_reload_theme_clicked()
   emit reload_theme();
 }
 
+void AOConfigPanel::on_theme_variant_index_changed(QString p_text)
+{
+  Q_UNUSED(p_text);
+  m_config->set_theme_variant(w_theme_variant->currentData().toString());

```

Comment:
`<default>` would still get added here.
I assume that's intentional, because of what we've talked about before, that `<` and `>` are illegal in Windows paths, correct?

Reply:
Yes.